### ISSUE TITLE
The translations on docs.docker-cn.com are not completed

### ISSUE THREAD

**Issue by twang2218** [docker/docker.github.io#4574]
### Problem description

I'm very excited that Docker has officially supported a Chinese version of the docker documents, <https://docs.docker-cn.com>, it's huge help for Chinese developers and ops. Thank you.

However, I found many documents are not fully translated, such as:

* https://docs.docker-cn.com/engine/userguide/

I understand that there might be some resource limitations here. What can I do to help the translation?

**Comment by twang2218** [comment]
Is anyone here can give some response? Did I raise the issue in the wrong repo?

**Comment by thaJeztah** [comment]
ping @johndmulhausen

**Comment by twang2218** [comment]
We have identified several typos in the docker-cn website, such as:

https://docs.docker-cn.com/get-started/part2/#dockerfile

```Dockerfile
ADD ./app
```

should be

```dockerfile
ADD . /app
```

And https://docs.docker-cn.com/get-started/part3/#docker-composeyml

```yaml
version:"3"
services:
  web:
    # 将 username/repo:tag 替换为您的名称和镜像详细信息
    image: username/repository:tag
    deploy:
      replicas:5
      resources:
        limits:
          cpus:"0.1"
          memory:50M
      restart_policy:
        condition: on-failure
    ports:
      - "80:80"
    networks:
      - webnet
networks:
  webnet:
```

It missed all the spaces between keys and values.

These have caused many troubles for people learning Docker already. If there is a way we can contribute the translation, then it would be great to minimize the trouble, and it would be also very helpful for Chinese docker users to contribute back to the community.

Thanks.

**Comment by johndmulhausen** [comment]
There are two things that I have proposed doing to help: 

1. I have signed off a Statement of Work clearing our translation vendor to translate the entire site. We have a quote from them to do this that seemed reasonable to me. 

2. We are in talks with another vendor to help us open-source the Chinese docs so that they can benefit from corrections from the community. We would host the repo and they would maintain it in the proposal as outlined. 

The proposal could be rejected as it does cost the company money to do these things but I believe in them and I know they would help improve things immensely. Will let this space know when there are answers.
